docs/execution-providers/MIGraphX-ExecutionProvider.md

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
There is already a page for this EP here: https://onnxruntime.ai/docs/execution-providers/community-maintained/MIGraphX-ExecutionProvider.html

Can/should it be consolidated?

Reply:
I added this to mirror what was done for the ROCm EP documentation.  Should we be removing the other then is what you're saying?

Reply:
Consolidated info to this document page instead and removed the one under community-maintained.